### DISCUSSION TITLE
The Actions UI does not properly update in many areas

### DISCUSSION THREAD

**Discussion by flyx** [community/community#102776] [Actions]
### Select Topic Area

Bug

### Body

I started using GitHub Actions some weeks ago and noticed some problems with the UI not updating properly in various areas. Some issues even got introduced recently.

## Action Status on Repository Page

When I view a repository and the most recent commit has triggered an action, there's an orange circle I can click to see details on the action. Since a week or two, this doesn't work properly anymore, I see a spinner for forever:

<img width="644" alt="Screenshot 2024-01-30 at 16 20 23" src="https://github.com/community/community/assets/791710/a1ecd4c7-5c8a-4d3e-bb6e-51aba6b62519">

I can close it and click again, and suddenly the action is listed properly:

<img width="642" alt="Screenshot 2024-01-30 at 16 27 16" src="https://github.com/community/community/assets/791710/569e6c23-e93d-41bf-a7ee-f64953f7232d">

Also, when the action succeeds, the orange circle doesn't update to a green checkmark until I refresh the page.

This worked previously (~2 weeks ago).

## Job Status

When I click on the running job, I see the logs updating in real time. However when the job finishes, while the steps will be marked as completed, the job is not: (And the counter below **„Create Release“**  continues counting up for all eternity) 
 
<img width="475" alt="Screenshot 2024-01-30 at 16 29 59" src="https://github.com/community/community/assets/791710/cff88a9a-84aa-4b72-aa48-275130330174">

Again, I need to refresh to see that the job is finished. This also happens when the job fails.

## Action Summary

In the summary, the job status is also not updated when it finishes, the counter keeps counting and the status keeps being „in progress“:

<img width="604" alt="Screenshot 2024-01-30 at 16 44 02" src="https://github.com/community/community/assets/791710/6d162ba3-66ed-4a12-accc-da4d30ef136a">

## Workflow Dispatch

When a workflow has a `workflow_dispatch` trigger and I trigger it manually, the newly triggered run is not visible *even though* the page refreshes. I need to refresh it *again* to see the workflow run.

## Summary

When a job finishes, various views in the UI do not update with this information, which makes the UI unfit for monitoring progress. I should not need to refresh the page to see that a job finished, particularly in the summary view.

**Comment by pichu1995** [comment]
I'm facing this same issue, It's not working even after the hard refresh. What is the fix for this.
